src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -138,7 +138,8 @@ public static void main(final String[] args) {
         IntervalList intervals = null;
         SAMSequenceDictionary intervalsSamSequenceDictionary = null;
         if (usingIntervals) {
-            log.info("Loading up region lists.");
+            USE_VCF_INDEX = true;   // If intervals are specified, the VCF files must be indexed
```

Comment:
I would suggest put this in the custom command line validation or emit a log warning stating you are setting ```UCE_VCF_INDEX```.